### PR TITLE
Update debug declaration to add destroy function

### PR DESCRIPTION
**Pull request by brandonros**
It's a one line PR change, I'm not spending 30 minutes filling out a questionnaire...

Add destroy typing

Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by Qix-** [comment]
> It's a one line PR change, I'm not spending 30 minutes filling out a questionnaire...

You spend about 10 seconds. I'm sure it took you longer to type that sentence.

The trade-off for when considerate authors of PR's choose to be decent contributors and fill out the form is that is significantly reduces the time necessary to triage and review a PR.

However, twice now you've chosen to be pushy and entitled in your comments on this site, first to me and now to the hard working people here that maintain DT, ultimately to the detriment of my time and the time of the people who look over these PRs.

Please check yourself, and edit your original post to fill out the very short summary of information asked of you.

Thank you.

File: types/debug/index.d.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ declare namespace debug {
         enabled: boolean;
         log: (...args: any[]) => any;
         namespace: string;
+        destroy: () => void;
```
Comment (uniqueiniquity) [inline types/debug/index.d.ts]: From the [library's source](https://unpkg.com/debug@4.1.1/dist/debug.js), it appears that this actually returns `boolean`, not void.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]
Since this is a popular package, please add a test ensuring your change doesn't get regressed in future updates. Thank you!

**Comment by brandonros** [comment]
> Please check yourself, and edit your original post to fill out the very short summary of information asked of you.

Good time spending more time writing a long winded post trying to prove how smart and cool you are to the open source zealots instead of contributing the time.

**Comment by brandonros** [comment]
> Since this is a popular package, please add a test ensuring your change doesn't get regressed in future updates. Thank you!

I don't meany any disrespect, but I'm not going to invest the time to learn a foreign package's test setup for a 1 line change. Somebody else can do this. You can close the PR if you want. I'm trying to give as little as possible. I don't know where the mindset "it's somebody else's problem" comes from, but... it's a one line change guys... come on.

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Review by uniqueiniquity — APPROVED** [review]

**Comment by Qix-** [comment]
@brandonros The posts above are easily the most entitled, toxic posts I've ever seen on Github. I really hope the trajectory of the community does not follow suit. I hope you reconsider your attitude and personality.

**Pull request merged**